tests/phpunit/WikiManagerTest.php

```diff
@@ -35,8 +40,13 @@ public function testCreate() {
 
 		$wikiManager = new WikiManager( 'createwikitest' );
 
+		$this->assertInstanceOf( WikiManager::class, $wikiManager );
```

Comment:
What's the reasoning behind this test? There shouldn't be any reason at all this wouldn't match is there?

Reply:
I intended to test the constructor itself, like some other WMF's extensions.[1] But I had to introduce WikiManagerTest::testConstruct() instead inserting code to existing WikiManagerTest::testCreate().
If your question is why constructors should be tested, I've thought and searched about it now, I haven't found the answer. Maybe because some calls to `new` make an exception raise? If you don't think these are needed, I will remove these in the next commit.

1: https://gerrit.wikimedia.org/g/mediawiki/extensions/AbuseFilter/+/bb3232ca72796586dc26b49618ebb7f5f6b6ca31/tests/phpunit/unit/CentralDBManagerTest.php#22

Reply:
To minimize the conflict, I've removed the addition to the test class as I think `class_alias` works fine for the last test.

Reply:
Thanks!